docs/adr/adr-006-fraud-service.md

```diff
@@ -188,20 +193,18 @@ Both full and light nodes should stop `DAS`, `Syncer` and `SubmitTx` services.
 
 4. Valid BadEncodingFraudProofs should be stored on the disk using `FraudStore` interface:
 
+### BEFP storage
+BEFP storage will be created on first subscription to Bad Encoding  Fraud Proof.
+BEFP will be stored in datastore once it will be received, using `fraud/badEncodingProof` path and a block Hash as a key:
 ```go
-type FraudStore interface {
-   // Put stores given Proof by header's hash in respective for this fraud proof directory
-   Put(path FraudProofType, headerHash string, p Proof) error
-   // Get retrieves Proof by header's hash from respective for this fraud proof directory
-   Get(path FraudProofType, headerHash string) (Proof, error)
-   // GetMany retrieves all Proofs from the respective for this fraud proof directory
-   GetMany(path FraudProofType) ([]Proof, error)
-}
-
-type fraudstore struct{
-   path string
-}
+//  put adds a Fraud Proof to the datastore with the given key.
+func put(ctx context.Context, datastore.Datastore,datastore.Key,[]byte) error
+```
+Once a node starts, it will check if datastore has a BEFP:
+```go
+func getAll(ctx context.Context, ds datastore.Datastore) ([][]byte, error)
 ```
+In case if response error will be empty(not ```datastore.ErrNotFound```), then BEFP has been already added to storage and node should be halted.
```

Comment:
```suggestion
In case if response error will be empty (and not ```datastore.ErrNotFound```), then a BEFP has been already added to storage and the node should be halted.
```